rfcs/README.md

```diff
@@ -0,0 +1,28 @@
+# Saluki RFCs
+
+This directory contains all Saluki RFCs (Request for Comment), which are documents intended to provide insight into the
+design decisions and changes made to the Saluki project, and preserve them for future reference.
+
+## Writing an RFC
+
+Writing an RFC should not be an overly-complicated affair, because ultimately we want to focus on finding a solution to
+the stated problem. However, we all need to make sure we're on the same page when it comes to what an RFC should look like.
+
+Here are the high-level guidelines you should when writing an RFC:
```

Comment:
👍 these seem like a good start. I wouldn't mind seeing a template in the future.